### ISSUE TITLE
Segmentation fault with `declare` statement.

### ISSUE THREAD

**Issue by jails** [facebook/hhvm#7297]
### HHVM Version
3.14.5
(But works fine with version <= 3.10)

### Standalone code, or other way to reproduce the problem

```php
<?php
declare(ticks = 1) {
    $closure = function () {
    };
}
```

### Expected result
None

### Actual result
Core dumped: Segmentation fault
Stack trace in /tmp/stacktrace.XXXX.log

**Comment by aorenste** [comment]
I can repro:
hhvm: buck-out/gen/hphp/runtime/headers#default,headers/hphp/runtime/vm/func-emitter-inl.h:93: HPHP::Id HPHP::FuncEmitter::lookupVarId(const HPHP::StringData*) const: Assertion `hasVar(name)' failed.

**Comment by jails** [comment]
Thanks a bunch :100:

**Comment by jails** [comment]
Did someone know when this patch will be shipped in ?
Looks like the patch has not been included yet in the last release, the binary still continue to segfault.

**Comment by aorenste** [comment]
@jails We had some internal issues with our release system - this should have shipped in 3.16 - but we accidentally missed that release.

**Comment by jails** [comment]
Ok I understand now, thank you for the head up!
